### DISCUSSION TITLE
Streak 🔥 grace period/streak protection

### DISCUSSION THREAD

**Discussion by faizan-official** [dailydotdev/daily#1298] [Feature Requests]
I'm a streak 🔥 guy but I forget sometimes, and it get lost. I just get upset when the long number turns into 0 🥲. Wouldn't it be good if you can add grace period of 1 day may be or streak protection to restore streak (max 3 to 5 times / month) just like Snapchat? 🤔

**Comment by marahim20** [comment]
I agree. This would be great!
